content/docs/setup/kubernetes/install/kubernetes/index.md

```diff
@@ -7,20 +7,25 @@ aliases:
     - /docs/setup/kubernetes/quick-start/
 ---
 
-Follow this guide to quickly evaluate Istio in a Kubernetes cluster on any platform.
-This guide installs Istio's built-in **demo**
-[configuration profile](/docs/setup/kubernetes/additional-setup/config-profiles/)
-using basic Kubernetes commands without needing to download or install [Helm](https://github.com/helm/helm).
+Visit our [getting started guide](/docs/setup/kubernetes/getting-started/) to
```

Comment:
Wrong document. This is supposed to be added at the top of `content/docs/setup/kubernetes/_index.md`

Reply:
Just to make sure, you want it to link to the kubernetes installation landing page?